go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/onsi/ginkgo/v2 v2.17.1
 	github.com/onsi/gomega v1.32.0
 	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/rs/zerolog v1.33.0
```

Comment:
This dependency doesn't seem like it is used in the current branch code - maybe run `go mod tidy` again? At least on my machine that dropped it from the dependencies.

Reply:
Will remove. Added for debugging